js/client-teambuilder.js

```diff
@@ -592,6 +592,7 @@
 					buf += '<div class="setmenu setmenu-left"><button name="undeleteSet"><i class="fa fa-undo"></i> Undo Delete</button></div>';
 				}
 				buf += '<div class="setchart"><div class="setcol setcol-icon" style="background-image:url(' + Tools.resourcePrefix + 'sprites/bw/0.png);"><span class="itemicon"></span><div class="setcell setcell-pokemon"><label>Pokemon</label><input type="text" name="pokemon" class="chartinput" value="" /></div></div></div>';
+				buf += '<div class="setmenu"><button name="importSet"><i class="fa fa-upload"></i>Import</button></div>';
```

Comment:
I just tested this here and it seems the `setmenu` for deletedSet always gets displayed on the left, while the import is always displayed on the right.

I may have misunderstood what you said, but I wasn't able to spot any bugs with this.